addon/templates/components/lt-body.hbs

```diff
@@ -1,8 +1,14 @@
 <div id={{concat tableId '_inline_head'}} class="lt-inline lt-head"></div>
 
+{{#lt-scrollable 
+  autoHide=autoHideScrollbar
+  scrollEnd="onScrolledToBottom"
+  scrollBuffer=scrollBuffer
+  as |scrollbar|
+}}
 <table>
   <tbody class="lt-body">
-    {{#each rows as |row|}}
+    {{#each (compute (pipe scrollbar.update) rows) as |row|}}
```

Comment:
This is pretty confusing and will be for anyone that attempts to read this. Can you add a brief comment?